lib/client/api.go

```diff
@@ -1543,13 +1548,13 @@ func (tc *TeleportClient) SCP(ctx context.Context, args []string, port int, flag
 				return trace.Wrap(err)
 			}
 
-			err = client.ExecuteSCP(cmd)
+			err = client.ExecuteSCP(ctx, cmd)
 			if err != nil {
-				return onError(err)
+				exitErr = err
```

Comment:
Only saves the last exit error but it should be ok. Not breaking out of the loop for multiple files follows the existing behavior of OpenSSH `scp` - let me know if that's not desired.

Reply:
Perhaps the errors could be logged s.t. `tsh scp --debug` shows intermediate errors as well?  Potentially useful.

Reply:
I decided to roll back to old behavior of exiting upon first encountered error as it seems to make more sense.